### ISSUE TITLE
Submit qualitative assessments per crop

### ISSUE THREAD

**Issue by jgaehring** [runrig-coop/rgo-early-warning-system#7]
To provide more granularity and insight into specific crops based on how soon they will be need to be ready for pending orders, we will want to enable qualitative assessments for each crop individually. This will also require computing the overall status of the farm by averaging that status over multiple crops, with greater weight given to certain crops based on when they are due for pickup etc.

It occurs to me now that this will also enable us to display a status on one particular crop or order _across all farms_, which could be especially helpful, but we will probably have to wait on implementing that til we get a little further down the road, and should probably be a separate issue.
